### PR TITLE
Get screenshot and snapshot using selenium and remotewebdriver

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/logcollector.py [hunk 1]
```diff
@@ -27,9 +30,14 @@ class CollectingNode():  # pylint: disable=too-few-public-methods
     def __init__(self, name, ssh_login_info=None, instance=None, global_ip=None):
         self.name = name
         self.remoter = RemoteCmdRunner(**ssh_login_info) if ssh_login_info else None
+        self._ssh_login_info = ssh_login_info
         self._instance = instance
         self.external_address = global_ip
 
+    @property
+    def ssh_login_info(self):
+        return self._ssh_login_info
```
Comment (fruch) [inline sdcm/logcollector.py]: why not just name it ssh_login_info ? with no underscore ?

File: sdcm/cluster.py [hunk 2]
```diff
@@ -3836,6 +3846,7 @@ def configure_alert_manager(self, node):
     def restart_scylla_monitoring(self, sct_metrics=False):
         for node in self.nodes:
             self.stop_scylla_monitoring(node)
+            self.stop_selenium_remote_webdriver(node)
```
Comment (fruch) [inline sdcm/cluster.py]: why do we need to stop start it ? with monitoring ?

Comment (aleksbykov) [reply sdcm/cluster.py]: Don't need. Missed it. thanks.

File: sdcm/logcollector.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/logcollector.py]: yep. you right. Just repeated same as on BaseNode

File: docker/env/version [hunk 3]
```diff
@@ -1 +1 @@
-0.57
+0.58
```
Comment (fruch) [inline docker/env/version]: bump it to 0.59, 0.58 is merged already today

Comment (aleksbykov) [reply docker/env/version]: fixed. and rebased on latest master

File: sdcm/cluster.py [hunk 4]
```diff
@@ -411,6 +412,10 @@ def short_hostname(self):
                 return "no_booted_yet"
         return self._short_hostname
 
+    @property
+    def ssh_login_info(self):
+        return self._ssh_login_info
```
Comment (fruch) [inline sdcm/cluster.py]: I think it would be better to replace all `_ssh_login_info` to `ssh_login_info`

File: sdcm/utils/remotewebbrowser.py [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+        self._remote_browser.get(url)
+        start_time = time.time()
+        try:
+            LOGGER.debug('Wait page loading ...')
+            WebDriverWait(self._remote_browser, self.load_page_timeout * 2).until(wait_page_loading)
+        finally:
+            LOGGER.debug('Create snapshot')
+            self._remote_browser.get_screenshot_as_file(screenshot_path)
+
+    def __del__(self):
```
Comment (fruch) [inline sdcm/utils/remotewebbrowser.py]: don't use __del__ for cleanups like that. you have little control on if it called at all.

I would do that explicitly in `get_screenshot` in a finally clause.

File: sdcm/cluster.py (hunk 4 above)
Comment (aleksbykov) [reply sdcm/cluster.py]: done

File: sdcm/utils/remotewebbrowser.py (hunk 5 above)
Comment (aleksbykov) [reply sdcm/utils/remotewebbrowser.py]: removed.

**Pull request merged**